### ISSUE TITLE
What's the state of this project?

### ISSUE THREAD

**Issue by hsingh23** [michaelficarra/CoffeeScriptRedux#361]
Is this the recommended version of coffescript? will it replace the current coffeescript? will there be more releases? Is there a maintainer for this project?

**Comment by deltaidea** [comment]
See #336.

> Anyone is free to work on this project. I stopped working on it after a disagreement between myself and the other CoffeeScript maintainers [...]. I don't see a future for the CoffeeScript language without adoption of ES6 features. Unfortunately, I was the only one that believed we should adopt the semantics of any equivalent features in ES6 [...]. So because I no longer use the language for my own projects, I am not motivated to work on it. But there's nothing stopping anyone from opening issues and sending PRs.

TL;DR: It's dead.

**Comment by forivall** [comment]
w.r.t adopting ES6 features, I'll be announcing something in a month or few.

**Comment by remoteportal** [comment]
(I hope CoffeeScript doesn’t die—it’s by far my favorite language right now.)

Peter Alvin
